### DISCUSSION TITLE
Put a pinned window behind others

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#1900] [Q&A]
I am trying to set up the GLava audio spectrum to be shown in every workspace behind the other windows. Would this be possible in any way? Thanks.

**Comment by vaxerski** [comment]
No, why would you want to do that?

**Comment by Aqa-Ib** [comment]
I like using audio spectrum visualizers, so I pin it to show in every workspace. Sometimes I would want to put a window in front of it and I could still be able to see it through window transparency. 
Waybar has this behaviour built-in though, configuring it as "layer: bottom". I don't know if I could launch an audio spectrum visualizer such as Cava or GLava as layer bottom too ?

**Reply by vaxerski** [reply]
nope, for xorg you have xwinwrap, but I am not aware of any such app for wayland.

**Reply by Aqa-Ib** [reply]
Thanks, we appreciate a lot your attention vaxerski.
